packages/RawButton/src/RawButton.js

```diff
@@ -86,7 +96,7 @@ const RawButton = React.forwardRef((props, ref) => {
       onKeyDown={handleKeyDown}
       onKeyUp={handleKeyUp}
       ref={rawButtonRef}
-      tabIndex={isDisabled ? null : tabIndex}
+      tabIndex={getTabIndex()}
```

Comment:
Changing so that if there is a tabIndex provided on the component it uses it, rather than the `disabled` prop overriding it. In the case of a menubar we want even disabled items to be focusable.